_data/people/Pretty codes

```diff
@@ -0,0 +1 @@
+writeprettycodes.heygal
```

Comment:
@Chloe-python-namibia: Kudos on opening your first pull request. This file needs more info before we can merge it, but we're happy to guide you through the process. For starters, you'll need to remove the existing content, and replace it with your username, name, etc., so that the format looks like [this example](https://github.com/python-namibia/python-namibia.github.io/blob/e34f9695a8a7bb8d337563781317fa835c115111/_data/people/octocat.yml). (If you follow that example, be sure to replace the example data with *your* data. For example, instead of `username: octocat`, you'd put `username: Chloe-python-namibia`, and so on.)

If you have any questions, or you need any help, just leave a comment here to let us know.